### PR TITLE
About Name component

### LINKED ISSUE #7: About "Name"
image

Just have a component with the text, don't worry about the graphics in the background

### PR DESCRIPTION
**Pull request by d-leeee**
Added tailwind custom configs
Added About Name component
Screenshot 2024-10-08 at 11 53 31 PM

### CODE REVIEW

**Comment by d-leeee** [comment]
Screenshot 2024-10-08 at 11 53 31 PM

File: src/components/about.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+interface AboutProps {
+  name: string;
+  description: string;
+}
```
Comment (stanleylew5) [inline src/components/about.tsx]: you're passing in props here, but we actually don't need props passed in here haha should've specified in the issue description. we'll have some actual text on the figma there soon. for now you can just put the lorem ipsum text in

File: src/components/about.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const About = ({ name, description }: AboutProps) => {
+  return (
+    <div className="flex flex-col items-center text-center font-Malayalam font-normal">
+      <div className="text-7xl">About</div>
```
Comment (stanleylew5) [inline src/components/about.tsx]: use vh, vw units instead ex: text-[5vw] here's an example of what they are

https://www.sitepoint.com/css-viewport-units-quick-start/

We'll go over this at tomorrow's meeting

File: src/components/about.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      <div className="pb-20 text-9xl">{name}</div>
```
Comment (stanleylew5) [inline src/components/about.tsx]: same thing as above comment

File: tailwind.config.ts [hunk 4]
```diff
@@ -2,7 +2,18 @@
...
   theme: {
-    extend: {},
+    extend: {
+      fontSize: {
+        description: ["25.26px", "35.98px"],
+      },
+      lineHeight: {
+        185: "185px",
+        87: "87px",
+      },
```
Comment (stanleylew5) [inline tailwind.config.ts]: let's not hardcode in values for font size and line height.  we'll go more in depth into this tomorrow but if you don't want to wait til tomorrow try experimenting with vh vw, % units.

**Review by stanleylew5 — CHANGES_REQUESTED** [review]
About.tsx, and place it within a folder within the components folder called home. you'll need to create the home folder

File: src/components/about.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+interface AboutProps {
+  name: string;
+  description: string;
+}
+
+const About = ({ name, description }: AboutProps) => {
+  return (
+    <div className="flex flex-col items-center text-center font-Malayalam font-normal">
```
Comment (stanleylew5) [inline src/components/about.tsx]:
```suggestion
    <div className="flex flex-col items-center text-center font-Malayalam">
```
not needed, does nothing

**Review by stanleylew5 — CHANGES_REQUESTED** [review]

**Pull request merged**